roles/hosted_engine_setup/tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -56,6 +56,14 @@
   - name: Import OpenSCAP task
     import_tasks: ../apply_openscap_profile.yml
     when: he_apply_openscap_profile|bool
+  - name: Enable FIPS on the engine VM
+    block:
+      - name: Enable FIPS
+        command: >-
+          fips-mode-setup --enable
+        changed_when: true
+      - import_tasks: ../reboot_to_enable_fips.yml
+    when: not he_apply_openscap_profile|bool and he_enable_fips|bool
```

Comment:
I don't like this very much. I'd rather see a single place doing the reboot (although np keeping in an include file) - either explicitly 'when: he_apply_openscap_profile|bool or he_enable_fips|bool' or 'when: need_to_reboot_he_vm' and set this var where/as needed.